run_dist.sh

```diff
@@ -1,4 +1,4 @@
-export CUDA_VISIBLE_DEVICES=4,5,6,7
+#export CUDA_VISIBLE_DEVICES=4,5,6,7
 PORT=${1:-29501}
 NGPU=${NGPU:-"4"}
 LOG_RANK=${LOG_RANK:-0,1,2,3}
```

Comment:
nit: maybe we should remove this script and just keep an example in a document?

Reply:
hmm, i use it like 50-150x a day during development so I don't want to remove it :) 
the export part is to keep the main gpus free for others that share my server.